src/vma/dev/ring_tap.cpp

```diff
@@ -1161,10 +1159,23 @@ void ring_tap::prepare_flow_message(vma_msg_flow& data, msg_flow_t flow_action,
 		data.flow.t5.src_ip = flow_spec_5t.get_src_ip();
 		data.flow.t5.src_port = flow_spec_5t.get_src_port();
 	}
+
+	rc = g_p_agent->send_msg_flow(&data);
+
+	/* Duplicate rules against lo interface */
+	if (0 == rc) {
+		data.if_id = 1;
+
+		rc = g_p_agent->send_msg_flow(&data);
+	}
+
+	return rc;
```

Comment:
Sending two diffrent message will increase the duration of each attach_flow...
i think we should modify the message to include both interfaces...
Are you sure about the index number ?